apps/test/unit/templates/manageStudents/ManageStudentsTableTest.js

```diff
@@ -298,6 +320,18 @@ describe('ManageStudentsTable', () => {
       expect(loginInfo.find('SignInInstructions').props().loginType).to.equal(
         SectionLoginType.google_classroom
       );
+      expect(wrapper.containsMatchingElement(<NoSectionCodeDialog />)).to.equal(
+        true
+      );
+      expect(
+        wrapper.find('NoSectionCodeDialog').props().typeClassroom
+      ).to.equal(SectionLoginType.google_classroom);
+      expect(wrapper.find('NoSectionCodeDialog').props().isOpen).to.equal(
+        false
+      );
+      const mainTable = wrapper.find('ManageStudentsTable');
+      mainTable.setState({showSectionCodeDialog: true});
+      expect(wrapper.find('NoSectionCodeDialog').props().isOpen).to.equal(true);
```

Comment:
Unit tests should generally test one thing. In this case (and in `renders correctly if loginType is clever`) I'd recommend splitting the render unit test from the "dialog opens" unit test. That way, when something fails, it's clear which part failed. Additionally, it keeps earlier failures in the test from hiding later failures.